onboarding_project/squad_pantry_app/models.py

```diff
@@ -0,0 +1,55 @@
+from django.db import models
+from django.utils import timezone
+from django.contrib.auth.models import User
+from django.core.validators import MinValueValidator
+
+
+class Dish(models.Model):
+    dish_name = models.CharField(max_length=256, unique=True)
+    NON_VEG = 0
+    VEG = 1
+    EGG = 2
+    DISH_TYPE = (
+        (NON_VEG, 'Non-Vegetarian'),
+        (VEG, 'Vegetarian'),
+        (EGG, 'Contains Egg'),
+    )
+    dish_type = models.IntegerField(choices=DISH_TYPE, default=NON_VEG)
+    is_available = models.BooleanField(default=False)
```

Comment:
help_text. `is_available` might not be intuitive enough for the user to know how to use this field